lib/import.php

```diff
@@ -188,6 +188,11 @@ protected function blob_to_post( WordPress_GitHub_Sync_Blob $blob ) {
 
 			if ( array_key_exists( 'post_date', $meta ) ) {
 				$args['post_date'] = $meta['post_date'];
+
+				if ( empty( $meta['post_date'] ) ) {
+					$meta['post_date'] = current_time( 'mysql' );
+				}
```

Comment:
I think this block should go before we assign the value of `$meta['post_date']` to `$args['post_date']`, so args gets the correct value for both `post_date` and `post_date_gmt`.

Reply:
😄 I paid zero attention that `$args` was used below, this is why I ❤️ code reviews!

Reply:
😄 completely ignored the fact `$args` was used below, will get that first thing in the am for ya - this is why I ❤️ code reviews!